ide-configuration/intellij-configuration/code-style/intellij-code-style_droolsjbpm-java-conventions.xml

```diff
@@ -1,12 +1,11 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<code_scheme name="Drools and jBPM: Java Conventions">
-  <option name="LINE_SEPARATOR" value="&#10;" />
-  <option name="USE_FQ_CLASS_NAMES_IN_JAVADOC" value="false" />
+<code_scheme name="KIE Java Conventions">
+  <option name="LINE_SEPARATOR" value="&#xA;" />
   <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99999" />
   <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="1" />
   <option name="PACKAGES_TO_USE_IMPORT_ON_DEMAND">
     <value>
       <package name="org.junit.Assert" withSubpackages="true" static="true" />
+      <package name="org.mockito.Mockito" withSubpackages="true" static="true" />
```

Comment:
Can I ask that AssertJ be added, too? The same as the above, so should share the behavior.